pkg/api/configsync/v1beta1/helmconfig.go

```diff
@@ -59,17 +59,17 @@ type HelmBase struct {
 
 	// values to use instead of default values that accompany the chart. Format
 	// values the same as default values.yaml. These values will take precedence if
-	// used in conjunction with valuesFileSources. How to handle multiple valuesFiles is
+	// used in conjunction with valuesFileRefs. How to handle multiple valuesFiles is
```

Comment:
"if used in conjunction with valuesFileRefs" -> "if valuesFileRefs is also specified"